src/style.css

```diff
@@ -1 +1,78 @@
-/* Styles go here */
+/* Styles */
+
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+  font-family: "Roboto", "Montserrat", sans-serif;
+  font-size: 12px;
+}
+
+/* main header decoration: */
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  height: 60px;
```

Comment:
There is no need to hardcode container height, since you've already set contents height